src/status_im/contexts/wallet/add_account/create_account/key_pair_name/view.cljs

```diff
@@ -17,10 +17,22 @@
    :emoji        (i18n/label :t/key-name-error-emoji)
    :special-char (i18n/label :t/key-name-error-special-char)})
 
-(defn navigate-back
+(defn- navigate-back
   []
   (rf/dispatch [:navigate-back]))
 
+(defn- on-continue [workflow key-pair-name]
+  (case workflow
+    ;; TODO issue #19759. Implement creation account from private key
+    :import-private-key
+    (not-implemented/alert)
+
+    (:new-keypair :recovery-phrase)
+    (rf/dispatch [:wallet/generate-account-for-keypair
+                  {:keypair-name key-pair-name}])
+
+    (js/alert "Unknown workflow")))
```

Comment:
this unknown workflow thing is a bit misleading. Should we not log something more meaningful, even for developers to find it? 🤔